public/setup/atomicupdate/updatePath.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$sql[]="UPDATE `{$dbprefix}acces` SET `page` = '/absence' WHERE `nom`='Absences - Index';";
+$sql[]="DELETE FROM `{$dbprefix}acces` WHERE `page`= 'absences/voir.php';";
+$sql[]="UPDATE `{$dbprefix}acces` SET `page` = '/absence/add' WHERE `nom` = 'Absences - Ajouter';";
```

Comment:
Same than previous comment. Wehere page = . 
*But*, here, adding object/add is not necessary. You can just delete this route.